src/transport_performance/gtfs/validation.py

```diff
@@ -838,8 +853,12 @@ def _plot_summary(
         _check_column_in_df(df=summary_df, column_name=target_column)
         _check_column_in_df(df=summary_df, column_name=day_column)
 
-        # convert column type for better graph plotting
+        # convert column type for better graph plotting, use desc
+        ROUTE_LKP = get_saved_route_type_lookup()
```

Comment:
Make sure ROUTE_LKP[“route_type”] is cast to str too

Reply:
@r-leyshon This should be resolved now with commit 14c2f26